client/i18n.js

```diff
@@ -2,16 +2,17 @@ import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
 import Backend from 'i18next-http-backend';
-import { enUS, es, ja } from 'date-fns/locale';
+import { enUS, es, ja, frCA } from 'date-fns/locale';
 
 const fallbackLng = ['en-US'];
-const availableLanguages = ['en-US', 'es-419', 'ja'];
+const availableLanguages = ['en-US', 'es-419', 'ja', 'fr-CA'];
 
 export function languageKeyToLabel(lang) {
   const languageMap = {
     'en-US': 'English',
     'es-419': 'Español',
-    'ja': '日本語'
+    'ja': '日本語',
+    'fr-CA': 'French'
```

Comment:
This should be French in French (which I believe is Français?).

I also notice that the locale code is `fr-CA` but the label is just "French"—does it make sense to distinguish this as Canadian French? If not, would it be better to label this locale as just `fr`?

Reply:
> This should be French in French (which I believe is Français?).

Good catch! I agree, it should be:
```suggestion
    'fr-CA': 'Français'
```

> does it make sense to distinguish this as Canadian French? If not, would it be better to label this locale as just fr?

I believe it makes sense because some word are more specific to this region. For example, the word "courriel" is the official word for "e-mail" in Quebec, Canada but may not be used in France.

> Do you have another person or people who could take a look at the translation?

I have one person in mind, I'll ask and report back :)

Thanks!

Reply:
Thanks for the update! I'm also happy to post on the Processing forum, social media, etc. if you would like others to review your translation, let me know!

Reply:
I am so sorry, it turns out that I am unable to find someone that has time to do the review. So yes, please feel free to do your post :)